sympy/combinatorics/tests/test_fp_groups.py

```diff
@@ -830,28 +830,6 @@ def test_subgroup_presentations():
-        " b_1**-2*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1*c_3*b_1**2*c_3,"
-        " b_1*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3*b_1,"
-        " b_1**-1*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1**2*c_3,"
-        " b_1**-1*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**-2,"
-        " b_1*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**-2*c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**2*c_3*b_1**2*c_3*b_1**-2*c_3**-1,"
-        " c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1"
-        "*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3, b_1**-1*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1**2*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1**2"
-        "*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1**-2, b_1**-1*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1*c_3*b_1**2"
-        "*c_3*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**-2*c_3**-1, b_1**-1*c_3**-1*b_1*c_3*b_1**2*c_3*b_1*c_3*b_1**2*c_3"
-        "*b_1**-2*c_3**-1*b_1**-1*c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3,"
-        " c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3*b_1*c_3*b_1**2"
-        "*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3*b_1, b_1*c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3**-1"
-        "*b_1**-2*c_3**-1*b_1**-1*c_3*b_1**2*c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3*b_1,"
-        " b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2"
-        "*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**-2*c_3*b_1**-2*c_3**-1*b_1**-1*c_3, b_1**-1*c_3**-1*b_1*c_3*b_1**2*c_3**-1"
-        "*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2"
-        "*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1"
-        "*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2"
-        "*c_3**-1*b_1**-2, b_1**-1*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2"
-        "*c_3**-1*b_1**2*c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3"
-        "*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3*b_1**2*c_3**-1*b_1"
-        "*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3*b_1**2*c_3**-1*b_1**-2"
-        "*c_3**-1*b_1**-1*c_3*b_1**-2))")
-    assert str(reidemeister_presentation(f, H)) == k
+    gens, rels = reidemeister_presentation(f, H)
+    assert str(gens) == "(b_1, c_3)"
```

Comment:
I'd have to extract the `FreeGroupElement`s `b_1` and `c_3` from somewhere, and the only place would be `gens` itself.